salt/states/file.py

```diff
@@ -3177,6 +3178,13 @@ def append(name,
     context
         Overrides default context variables passed to the template.
 
+    ignore_whitespace
+        .. versionadded:: 2015.8.4
+
+        Spaces and Tabs in text are ignored by default, one space or multiple
+        tabs are the same for salt. Set this option to ``False`` if you want to
+        change this behavior.
+
```

Comment:
It's unclear from this comment where whitespace is ignored, in the inserting content or when searching for the inserting content. Could you just add something like `Space and tabs in text are ignored by default when searching for the appending content, one space or multiple...`